SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.0.0   | :white_check_mark: |
+| 0.20.x  | :white_check_mark: |
```

Comment:
What do these refer to ? presumably versions of Node-RED - or is it this project ?

Reply:
I asked @kazuhitoyokoi to copy over the standard SECURITY.md file from the main repository. I forgot that had this table. I was more concerned with having the information about how to report security issue - this table can be removed from the non-core copies.

Reply:
I learned what is the SECURITY.md file. I removed the table from the file.